user_guide.md

```diff
@@ -296,14 +296,30 @@ unsecured endpoint by default. For a production deployment with SSL and authenti
 
 We treat each deployed model as a [component](https://ksonnet.io/docs/tutorial#2-generate-and-deploy-an-app-component) in your APP.
 
-Create a component for your model
+Create a component for your model located on cloud
 
 ```
 MODEL_COMPONENT=serveInception
 MODEL_NAME=inception
 MODEL_PATH=gs://kubeflow-models/inception
+MODEL_STORAGE_TYPE=cloud
```

Comment:
this is not required right?

Reply:
yes, now i delete this line and detect if `'modelStorageType'` is an attribute of `$.params`, so that when model is in `gs`, the user don't need to specify `MODEL_STORAGE_TYPE=cloud`